Tools/flatpak/setup-rpkgs/R/flatpakGeneratePkgsList.R

```diff
@@ -0,0 +1,60 @@
+# This file needs internet!
+
+# TODO: look at https://github.com/andrie/miniCRAN/issues/50#issuecomment-374624319
+# it would be nice to be able to outsource this to miniCRAN which has a method for adding local pkgs to a repository
+# the only downside is that it looks like miniCRAN is not actively maintained anymore.
```

Comment:
We could do that, but in hindsight, I'm not 100% sure how `miniCRAN` interacts with `renv` because that always tries to access GitHub if the DESCRIPTION specifies it in its remotes. Instead, I created a fake local GitHub website for renv by redirecting the URLs it wants to "download" to valid local files (in a way that both https://api.github.com/repos/jasp-stats/jaspAnova and https://api.github.com/repos/jasp-stats/jaspAnova/commits/master are valid files).

The nice thing about using something like `miniCRAN`, however, is that we could outsource creating a valid CRAN repository. For example, there are a few exceptions I had to fix manually (e.g., downloading recommended R packages in a separate step) and `miniCRAN` *should* fix that for us.

Reply:
Good to keep in mind then